plugins/janus_recordplay.c

```diff
@@ -2280,6 +2318,37 @@ janus_recordplay_frame_packet *janus_recordplay_get_frames(const char *dir, cons
 			offset += len;
 			continue;
 		}
+
+		if(data) {
+			/* Things are simpler for data, no reordering is needed: start by the data time */
+			gint64 when = 0;
+			bytes = fread(&when, sizeof(gint64), 1, file);
+			if(bytes < 1) {
```

Comment:
See note on your other PR. I'd rather invert the arguments there, so that we still check the bytes.

Reply:
Done.

Reply:
Done.